javalang/tokenizer.py

```diff
@@ -579,10 +582,12 @@ def error(self, message, char=None):
 
         message = u'%s at "%s", line %s: %s' % (message, char, line_number, line)
 
-        raise LexerError(message)
+        if not self.ignore_errors:
+            raise LexerError(message)
 
-def tokenize(code):
+def tokenize(code, ignore_errors=False):
     tokenizer = JavaTokenizer(code)
```

Comment:
Would you mind adding an `ignore_errors` argument to the `JavaTokenizer` constructor and passing `ignore_errors` on this line instead? That way the option is more clearly documented for folks using `JavaTokenizer` directly rather than through this method.

Reply:
Done.